crates/re_renderer/src/renderer/generic_skybox.rs

```diff
@@ -1,37 +1,78 @@
 use smallvec::smallvec;
 
 use crate::{
+    allocator::create_and_fill_uniform_buffer,
     draw_phases::DrawPhase,
     include_shader_module,
     renderer::screen_triangle_vertex_shader,
     view_builder::ViewBuilder,
     wgpu_resources::{
+        BindGroupDesc, BindGroupLayoutDesc, GpuBindGroup, GpuBindGroupLayoutHandle,
         GpuRenderPipelineHandle, GpuRenderPipelinePoolAccessor, PipelineLayoutDesc,
         RenderPipelineDesc,
     },
 };
 
 use super::{DrawData, DrawError, RenderContext, Renderer};
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum GenericSkyboxType {
+    DirectionalGradientDark = 0,
+    DirectionalGradientBright = 1,
+}
+
+mod gpu_data {
+    use crate::wgpu_buffer_types;
+
+    #[repr(C, align(256))]
+    #[derive(Clone, Copy, bytemuck::Pod, bytemuck::Zeroable)]
+    pub struct UniformBuffer {
+        pub background_type: wgpu_buffer_types::U32RowPadded,
+        pub _end_padding: [wgpu_buffer_types::PaddingRow; 16 - 1],
+    }
+}
```

Comment:
For anyone who knows that there is an option to have a solid skybox color it would be god to comment somewhere here how that works (I assume by not using the skybox at all, and relying on clear color?)